src/SortChain.php

```diff
@@ -42,6 +48,10 @@ public function keys($collection)
 
     public function __invoke($a, $b)
     {
+        if (empty($this->callables)) {
+            return static::$nullOperatorEquality;
```

Comment:
> That said, if you have a list of completely identical elements, does it really matter which order they get sorted? :grinning:

had the same thought and considered deleting my comment :) But:
It does not seem to be relevant on a list of equal integers but it may affect how objects, that have different field values but are considered equal dependent on some criteria are sorted.